releasenotes/notes/asm-event-tracking-sdk-d7e8516511c73589.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    ASM: Added two new functions for the Application Security Events Tracking API; `track_user_login` and `track_custom_event`.
```

Comment:
Feel free to also mention the out-of-the box ATO (Account TakeOver) protection that will be provided thanks to the user login events.